Cargo.toml

```diff
@@ -1,12 +1,14 @@
 [package]
-name = "rust-app"
-version = "0.2.1"
-authors = ["yhql"]
+name = "app-boilerplate-rust"
+version = "1.0.0"
+authors = ["yhql", "agrojean-ledger"]
 edition = "2021"
 
 [dependencies]
 nanos_sdk = { git = "https://github.com/LedgerHQ/ledger-nanos-sdk.git" }
 nanos_ui = { git = "https://github.com/LedgerHQ/ledger-nanos-ui.git" }
```

Comment:
should be updated to 
`ledger_device_sdk = { git = "https://github.com/LedgerHQ/ledger-device-rust-sdk.git" }
ledger_device_ui_sdk = { git = "https://github.com/LedgerHQ/ledger-device-rust-sdk.git" }`

Reply:
Done. Except I'm using the branch of the workspace which adds new UI gadgets. We need to merge it so I can remove the branch specification here.